apps/betterangels-backend/schema.graphql

```diff
@@ -756,6 +761,8 @@ type UserType {
   username: String!
   isOutreachAuthorized: Boolean
   organizationsOrganization: [OrganizationType!]
+  hasAcceptedTos: Boolean!
```

Comment:
not sure if this should have the `!`

Reply:
ya, it's a non-nullable field with a default